shellfoundry_traffic/test_helpers.py

```diff
@@ -6,13 +6,14 @@
 from typing import Optional, Tuple
 
 import yaml
-import xml.etree.ElementTree as ET
+import xml.etree.ElementTree as Element_Tree
```

Comment:
If you want to follow PEP 8 this should be ElementTree.=
However, ET is the convention for ElementTree.
See in https://docs.python.org/3.7/library/xml.etree.elementtree.html

Reply:
Changed to ET